compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -557,6 +584,117 @@ struct QuantizeWeights final : public luci::CircleNodeMutableVisitor<bool>
   }
 };
 
+/**
+ * @brief QuantizeConstInputs quantizes const input tensors
+ * @details Quantize values using min/max of const values
+ */
+struct QuantizeConstInputs final : public luci::CircleNodeMutableVisitor<bool>
+{
+  QuantizeConstInputs(loco::DataType input, loco::DataType output)
+      : input_type(input), output_type(output)
+  {
+  }
+
+  loco::DataType input_type;
+  loco::DataType output_type;
```

Comment:
Do they have to be `public`?

Reply:
I've changed the struct to the function (not to use visitor pattern).